Adapter/MemcachedCache.php

```diff
@@ -11,8 +11,16 @@
 
 namespace Sonata\CacheBundle\Adapter;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\MemcachedCache class is deprecated since version 2.4 and will be removed in 3.0.'
```

Comment:
I copied the base from here:

```php
<?php

/*
 * This file is part of the Sonata Project package.
 *
 * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

namespace Sonata\AdminBundle\Admin;

@trigger_error(
    'The '.__NAMESPACE__.'\Admin class is deprecated since version 3.1 and will be removed in 4.0.'
    .' Use '.__NAMESPACE__.'\AbstractAdmin instead.',
    E_USER_DEPRECATED
);

/**
 * NEXT_MAJOR: remove this class.
 *
 * @deprecated since version 3.1, to be removed in 4.0. Use Sonata\AdminBundle\AbstractAdmin instead
 *
 * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
 */
abstract class Admin extends AbstractAdmin
{
}
```